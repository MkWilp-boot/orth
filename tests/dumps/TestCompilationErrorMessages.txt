error: unknow token "o" in "repo/compilation_error.orth" at line: 2 colum: 4
error: unknow token "psdsa" in "repo/compilation_error.orth" at line: 2 colum: 6
error: unknow token "k" in "repo/compilation_error.orth" at line: 3 colum: 4
error: unknow token "999" in "repo/compilation_error.orth" at line: 3 colum: 6