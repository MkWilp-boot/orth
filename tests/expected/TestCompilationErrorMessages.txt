Undefined token/unknow token "o" in "repo/TestCompilationErrorMessages.orth" at line: 2 colum: 4